mrtt-ui/src/views/ManageOrganizationUsers.js

```diff
@@ -7,36 +7,81 @@ import { ButtonPrimary, ButtonSecondary } from '../styles/buttons'
 import { ContentWrapper, TitleAndActionContainer } from '../styles/containers'
 import { PageSubtitle, PageTitle } from '../styles/typography'
 import { TableAlertnatingRows, TdCenter, ThCenter } from '../styles/table'
+import { toast } from 'react-toastify'
+import { useAuth } from '../hooks/useAuth'
 import language from '../language'
+import LoadingIndicator from '../components/LoadingIndicator'
+import LoadingIndicatorOverlay from '../components/LoadingIndicatorOverlay'
 import USER_ROLES from '../constants/userRoles'
 
 const pageLanguage = language.pages.manageOrganizationUsers
 
 const ManageOrganizationUsers = () => {
+  const [isLoading, setIsLoading] = useState(true)
+  const [isSubmitting, setIsSubmitting] = useState(false)
   const [organizationName, setOrganizationName] = useState()
   const [organizationUsers, setOrganizationUsers] = useState([])
+  const { currentUser } = useAuth()
   const { organizationId } = useParams()
   const organizationUrl = `${process.env.REACT_APP_API_URL}/organizations/${organizationId}`
   const organizationUsersUrl = `${organizationUrl}/users`
 
+  const updateOrganizationUser = (userWithUpdatedInfo) => {
+    const organizationUsersWithUserRoleUpdated = organizationUsers.map((existingUserInfo) => {
+      return existingUserInfo.id === userWithUpdatedInfo.id ? userWithUpdatedInfo : existingUserInfo
+    })
+    setOrganizationUsers(organizationUsersWithUserRoleUpdated)
+  }
+
   useEffect(
     function loadApiData() {
-      Promise.all([axios.get(organizationUrl), axios.get(organizationUsersUrl)]).then(
-        ([
-          {
-            data: { organization_name }
-          },
-          { data: organizationUsers }
-        ]) => {
-          setOrganizationName(organization_name)
-          setOrganizationUsers(organizationUsers)
-        }
-      )
+      Promise.all([axios.get(organizationUrl), axios.get(organizationUsersUrl)])
+        .then(
+          ([
+            {
+              data: { organization_name }
+            },
+            { data: organizationUsers }
+          ]) => {
```

Comment:
What's going on here? Is it destructuring the API responses?
I'm assuming it's an API issue that one property name is snake case and one is camel case

Reply:
ya, you can get crazy with destructuring. Should I avoid this? 

Ya, api properties. Not in my realm of control.